lib/msal-core/src/telemetry/TelemetryManager.ts

```diff
@@ -36,7 +36,15 @@ export default class TelemetryManager {
 
     constructor(config: TelemetryConfig, telemetryEmitter: TelemetryEmitter) {
         // TODO THROW if bad options
-        this.telemetryPlatform = config.platform;
+        this.telemetryPlatform = {
+            sdk: "Msal.js",
```

Comment:
Can we please use a constant instead ( esp to port this for other libraries as needed)?

Reply:
Good call

Reply:
updated